Fix: documentation for per-task timeout (#79715)
ansible/ansible: docs/docsite/rst/network/user_guide/network_debug_troubleshooting.rst

  Suggestions to resolve:

>   Some modules support a ``timeout`` option.

Reviewer: ```suggestion
  Some modules support a ``timeout`` option, which is different to the ``timeout`` keyword for tasks.
```

>   Some modules support a ``timeout`` variable.

Reviewer 2: option or parameter, as this is assigned to the module itself, `ansible_command_timeout` is a variable, whicih can be declared anywhere where variables are supported `vars:`, `set_fact`, `include_vars` and inventory.

>   Some modules support a ``timeout`` variable.

Reviewer: ```suggestion
  Some modules support a ``timeout`` option.
```

>   

Reviewer 2: tempted to add (which is different from the tasks's ``timeout`` keyword)

Reviewer: @bcoca More detail never hurt the docs. How about "Some modules support a ``timeout`` option, which is different to the ``timeout`` keyword for tasks."

  Suggestions to resolve:
  

>   If the module does not support the ``timeout`` variable directly it can be set via the ``ansible_command_timeout variable

Reviewer: ```suggestion
  If the module does not support the ``timeout`` variable directly you can use the ``ansible_command_timeout`` variable instead.
```

Reviewer 2: ^ correct considering the 'network module debugging' context, but not true for modules in general

>   If the module does not support the ``timeout`` variable directly it can be set via the ``ansible_command_timeout variable

Reviewer: ```suggestion
  If the module does not support the ``timeout`` option directly, most networking connection plugins can enable similar functionality with the ``ansible_command_timeout``  variable.
```

Reviewer: @bcoca Does this seem OK? Any other changes needed?

Reviewer 2: ```If the module does not support the ``timeout`` option directly, most networking connection plugins can enable similar functionality via the ``ansible_command_timeout``  variable.```
